src/lib.rs

```diff
@@ -46,14 +46,15 @@
 )]
 #![allow(clippy::match_bool)]
 #![forbid(unsafe_code)]
-#![cfg_attr(all(feature = "nightly", doctest), feature(external_doc))]
-// we pass "--cfg docsrs" when building docs to add `This is supported on feature="..." only.`
+#![cfg_attr(feature = "nightly", feature(doc_cfg))]
+
+// we pass "--cfg docsrs" when building docs to add `This is supported on
+// feature="..." only.`
 //
 // To properly build docs of this crate run
 // ```console
 // $ RUSTDOCFLAGS="--cfg docsrs -Znormalize-docs" cargo +nightly doc --open --all-features
 // ```
-#![cfg_attr(all(docsrs, feature = "nightly"), feature(doc_cfg))]
```

Comment:
Shouldn't this comment be above `feature(doc_cfg)`?

Reply:
You're right.